gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/InstrumentatedGoogleHadoopFileSystem.java

```diff
@@ -271,6 +273,11 @@ public String getScheme() {
     return null;
   }
 
+  @Override
+  public void close() throws IOException {
+    setHttpStatistics();
```

Comment:
Do we need to capture this at close ? Looks like any invocations for stats would return 0 till we close the FS, which is counter intuitive